src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -352,7 +350,7 @@ public boolean isProtected() {
     /**
      * Writes all data in the format <key, serialized data>.
      */
-    public Map<String, String> serialize() {
+    public Map<String, String> getAsString() {
```

Comment:
I am not so happy with naming convention here (and below). After all, this returns no String as the name indicates. Could we make the convention a little more flexible to better describe the result, say: `getAsStringMap`, `getAsStringList`, etc.?